### PR TITLE
Port to Python 3

### PR DESCRIPTION
**Pull request by kiy4h**

### CODE REVIEW

**Comment by tchx84** [comment]
I tested this, simply wrote "table" on the word field to translate it from English to Spanish and got the following error:

```
Traceback (most recent call last):
  File "/home/sugar/words-activity/wordsactivity.py", line 615, in __totranslate_activated_cb
    self._translate()
  File "/home/sugar/words-activity/wordsactivity.py", line 641, in _translate
    translations[i] = translations[i].decode()
AttributeError: 'str' object has no attribute 'decode'
```

File: dictdlib.py [hunk 1]
```diff
@@ -59,6 +59,7 @@ def b64_decode(str):
         shiftval += 6
     return retval
 
+
```
Comment (tchx84) [inline dictdlib.py]: A typo?

**Review by tchx84 — CHANGES_REQUESTED** [review]
Please test the activity as well.

File: dictdlib.py (hunk 1 above)
Comment (kiy4h) [reply dictdlib.py]: Whoops, fixed.

**Comment by kiy4h** [comment]
I forget to test after f3f2527, it won't happen again. It was now tested in a VM and running smoothly.

**Comment by quozl** [comment]
Thanks.  Great progress.  Tests out fine for me.

However, you should have opened this pull request to merge into the _master_ branch.  I've taken your changes relative to @pro-panda's and pushed to _master_ along with my own changes.

**Comment by kiy4h** [comment]
Whoops, sorry!

**Comment by Kiy4h** [comment]
Whoops, sorry!

**Comment by Kiy4h** [comment]
Whoops, sorry!

**Comment by Kiy4h** [comment]
Whoops, sorry!

**Comment by quozl** [comment]
No worries, I can git.  :grin:

**Pull request closed without merging**